src/corehost/cli/deps_resolver.cpp

```diff
@@ -89,8 +63,41 @@ void add_unique_path(
     existing->insert(real);
 }
 
+pal::string_t get_deps_filename(const pal::string_t& path)
```

Comment:
Why is this named "get_deps_filename"?  Isn't it a utility to find any file name?

Don't we already have this utility somewhere?  [https://github.com/dotnet/core-setup/blob/88da334986d5f560c98248c2cfff6e5816cdde7a/src/corehost/common/utils.h#L26](https://github.com/dotnet/core-setup/blob/88da334986d5f560c98248c2cfff6e5816cdde7a/src/corehost/common/utils.h#L26)

Reply:
Note the hard-coded backslash for deps files. get_filename will use the platform-specific backslash\forwardslash

Reply:
Can you rename the method and/or add a comment to make that more obvious?

Reply:
I added a comment